dhcpv6/option_informationrefreshtime.go

```diff
@@ -0,0 +1,46 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"github.com/u-root/u-root/pkg/uio"
+	"time"
+)
+// OptInformationRefreshTime implements OptionInformationRefreshTime option.
+// https://tools.ietf.org/html/rfc8415
```

Comment:
nit: for a bit more accuracy
```suggestion
// https://tools.ietf.org/html/rfc8415#section-21.23
```